content/donate.md

```diff
@@ -1,10 +1,11 @@
 # Donate
 --------
 
-If you use makedeb or any of it's related products for your own use, please consider donating to help futher development. Your donation will help keep servers running, help pay for developer resources, and let the current developers know that what we're creating is found useful by you.
-Ways to donate
+If you use makedeb or any of its related products for your own use, please consider donating to help futher development. Your donation will help keep servers running, help pay for developer resources, and let the current developers know that what we're creating is found useful by you.
+
+### Ways to donate
```

Comment:
Can you change this to just `##`? I don't think I have any header styles defined for the variant with three, and I think other stuff like the news section primarily uses two as well.